pennylane/devices/experimental/device_api.py

```diff
@@ -173,7 +174,7 @@ def preprocess(
 
         """
 
-        def blank_postprocessing_fn(res):
+        def blank_postprocessing_fn(res: ResultBatch) -> ResultBatch:
```

Comment:
This line is not covered.

Reply:
Since this class is just an interface with only null behavior, everything should be tested in implementations, rather than here.

Reply:
Will soon be covered in #4111